src/main-client-v2.ts

```diff
@@ -3190,6 +3191,328 @@ export class MainClient extends BaseRestClient {
     return this.getPrivate('sapi/v1/pay/transactions', params);
   }
 
+  /**
```

Comment:
This is where new part  of broker API starts.  It is now called Exchange link, as you saw. 

Note: I added some of the missing endpoints, but only some. THere is 10 or so missing, I will take care of that later when we are done. with this rearranging.